application/controllers/HostController.php

```diff
@@ -88,6 +88,7 @@ public function sourceAction()
         }
 
         $this->addControl((new HostList([$this->host]))
+            ->setViewMode('common')
```

Comment:
If you're applying this here, you also have to apply it on the main lists. Which doesn't suit I guess, so please think of another solution. Possibly a CSS only solution.

Reply:
✅